specs/dynamicdependencies/DynamicDependencies.md

```diff
@@ -538,13 +542,13 @@ Packaged processes are a problem. The PATH environment variable and AddDllDirect
 
 None of these are sufficiently flexible to leverage for Dynamic Dependencies.
 
-**TODO** Perhaps the [uap7:ImportRediorectionTable](https://docs.microsoft.com/en-us/uwp/schemas/appxpackage/uapmanifestschema/element-uap7-importredirectiontable) supporting machinery can help out here?
+**TODO** Perhaps the [uap7:ImportRedirectionTable](https://docs.microsoft.com/en-us/uwp/schemas/appxpackage/uapmanifestschema/element-uap7-importredirectiontable) supporting machinery can help out here?
 
 ## 3.2. Known Issues for Packaged Processes
 
 ### 3.2.1. Known Issue: DLL Search Order for uap6:LoaderSearchPathOverride
 
-Packages can specify [uap6:LoaderSearchPathOverride](https://docs.microsoft.com/en-us/uwp/schemas/appxpackage/uapmanifestschema/element-uap6-loadersearchpathoverride) to add additional (package-relative) paths to the Loader's search order, and even potentially remove a package's root directory from the Loader's search order. If present the DLL Search Order for the package graph isn't the list of packages in the package graph and there are no public APIs to get this list of information. The only available mechanism is to `Find, Load and Parse appxmanifest.xml` to determine if this extension is present and, if so, honor it.
```

Comment:
Can you wordwrap everything at ~80? It's hard to tell what specifically changed, GitHub's diff is line-oriented.

Reply:
Easy enough for my editor. 80 or something bigger? Most styles and practices (formal and informal) I've seen in recent years go with 120 or 132, and maybe a soft guideline of 80